sktime/datatypes/_utilities.py

```diff
@@ -290,3 +290,69 @@ def get_window(obj, window_length=None, lag=0):
     raise ValueError(
         "bug in get_latest_window, unreachable condition, ifs should be exhaustive"
     )
+
+
+def get_slice(obj, start=None, end=None):
+    """Slice obj with start (inclusive) and end (exclusive) indices.
+
+    Returns time series or time series panel with time indices
+        strictly greater than cutoff - lag - window_length, and
+        equal or less than cutoff - lag.
+    Cutoff if of obj, as determined by get_cutoff.
+
+    Parameters
+    ----------
+    obj : sktime compatible time series data container or None
+        if not None, must be of one of the following mtypes:
+            pd.Series, pd.DataFrame, np.ndarray, of Series scitype
```

Comment:
this is also not correct, no?

Currently, slicing works only if `obj` is one of the `Series` types.

It will break if the `pd.DataFrame` that reaches line 342ff has a row `MultiIndex`.

You don't need to generalize this, but the docstring should be correct about the assumptions.